docs/manpages/zone2ldap.1.md

```diff
@@ -20,6 +22,15 @@ inserted to an LDAP server.
 --basedn=*DN*
 :    Base DN to store objects below
 
+--create-ent
```

Comment:
If you are using the "tree" layout, then ENTs will be created without this flag. However with other layouts it's not necessarily wanted by the LDAP admin, which is why I think it's best to leave this as a separate option.

Then maybe adding a blanket "--dnssec" flag, as @zeha, propsed, that would turn on the necessary flags is something that can be added.